### DISCUSSION TITLE
Update Logo

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3233] [Help]
Hi guys, is it possible to update BudiBase logo with our own logo ?

![image](https://user-images.githubusercontent.com/597759/140032004-7e8189ea-dfaa-4e96-bd1d-7a334010921c.png)

**Comment by mjashanks** [comment]
Yes - you need to be an administrator to do this. 

From the screen that you posted above:
1. Click on your profile icon, in the top-left
2. Click "Open developer mode"
3. Goto the `Organisation` tab.

In there, you can change your logo and company name.

**Reply by ryzam** [reply]
I did step that you provide but still no success. After click save button it show the setting is successful but when refresh the image was disappear.

![image](https://user-images.githubusercontent.com/597759/140034600-9e3627a0-4587-402c-b8b0-350c7182b27f.png)

![image](https://user-images.githubusercontent.com/597759/140034950-f8048872-c928-445d-98d0-d2fd9706aeb8.png)

**Reply by mjashanks** [reply]
Ah - are you using Budibase Cloud? It's working for me on my self-hosted instance, but not cloud. I will log an issue for this.

**Reply by marceldarvas** [reply]
![image](https://user-images.githubusercontent.com/10864884/152078610-2107c4ec-67fd-4611-ae41-be3a284838e3.jpeg)

I have a self-hosted instance, Org logo set that works in other screens but not on the user invite page.

**Reply by mjashanks** [reply]
Thanks @marceldarvas  - logged this here: https://github.com/Budibase/budibase/issues/4290
